src/ServicePulse.Host/app/js/views/failed_groups/controller.js

```diff
@@ -159,10 +160,26 @@
                 });
         };
 
+        var storeSelectedClassification = function (classification) {
+            $cookies.put("failed_groups_classification", classification);
+        };
+
+        var getDefaultClassification = function(classifiers) {
```

Comment:
maybe `load` instead of `get`? It's minor, but load to me implies getting data from somewhere, while get implies the data is available already.

Maybe get is actually better then. If we choose get, maybe the previous methods should also be changed to get

Reply:
I tend to keep the changes to a minumum so any refactoring I stay clear of. I'm happy to clean it up in a separate commit.